README.md

```diff
@@ -347,9 +378,33 @@ knows Staltz is part of the group because all `group/add-member` messages have
 Staltz can see Arj has invited Mix because he's replicating Arj's
 "group/additions" feed, so Staltz starts replicating Mix's group feed.
 
+## 5. Security Considerations
+
+TBD
+
+## 6. References
+
+### 6.1. Normative References
+
+- [ssb-meta-feeds-spec]
+- [envelope-spec]
+- [private-group-spec]
+- [ssb-meta-feeds-dm-spec]
+
+### 6.2. Informative References
```

Comment:
here is might be good to link to other (non-ssb) group systems. Such as MLS (https://messaginglayersecurity.rocks/). There should be some notes in our chat about them in decentralized settings. I can try and find them again and add here. Not that it should block this PR. The structure is great and we can always add more information.

Reply:
Agree, although I would want to put MLS reference (and a comparison discussion) in the "private groups spec" while this spec is just about the metafeeds+groups. I actually really like [original_notes.md](https://github.com/ssbc/envelope-spec/blob/master/original_notes.md) as a document and prefer it over the actual envelope-spec and private-group-spec, because it tells a story instead of being just bullet lists.

Reply:
Ah right. Make sense. Good we have the original notes :)